finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,31 @@
+for(var i = 1; i < 100; i++){
+  if(i % 15 === 0){
+    console.log("FizzBuzz")
+  } else if (i % 5 === 0){
+    console.log("Buzz")
+  } else if(i % 3 === 0){
+    console.log("Fizz")
+  } else {
+    console.log(i)
+  }
+}
+
+//this should work for any number combination you would want to plug in using the variables below the code
+function fizzBuzz(num, num2, numCombined) {
+  for(var i = 1; i < 100; i++){
+    if(i % numCombined === 0){
+      console.log("FizzBuzz")
+    } else if (i % num2 === 0){
+      console.log("Buzz")
+    } else if(i % num === 0){
+      console.log("Fizz")
+    } else {
+      console.log(i)
+    }
+  }
+};
+
+var num = 2;
+var num2 = 4;
+var numCombined = num * num2
```

Comment:
I don't think the parameter `numCombined` needs to be passed in as an argument to the `fizzBuzz` function.  You could calculate `num * num2` within the function, like so -

```
var num = 2;
var num2 = 4;
fizzBuzz(num, num2);

function fizzBuzz(num, num2) {
  var numCombined = num * num2;
  for(var i = 1; i < 100; i++){
    if(i % numCombined === 0){
      console.log("FizzBuzz")
    } else if (i % num2 === 0){
      console.log("Buzz")
    } else if(i % num === 0){
      console.log("Fizz")
    } else {
      console.log(i)
    }
  }
};
```

Reply:
@mschae16 that totally makes sense, thanks.